packages/cardano-graphql/src/operations/transactions.graphql

```diff
@@ -0,0 +1,303 @@
+fragment metadatumValue on Metadatum {
+  __typename
+  ... on BytesMetadatum {
+    bytes
+  }
+  ... on IntegerMetadatum {
+    int
+  }
+  ... on StringMetadatum {
+    string
+  }
+}
+
+fragment metadatumMap on MetadatumMap {
+  map {
+    label
+    metadatum {
+      ...metadatumValue
+      ... on MetadatumArray {
+        ...metadatumArray
+      }
+      # Review: nested maps and arrays are currently supported up to 1 level of nesting,
```

Comment:
Noted. It seems we need to dedicate some time to covering all the metadata permutations, but think it's fine to decouple from this PR